stout_a07b.js

```diff
@@ -0,0 +1,101 @@
+/*Kyle Stout
+  stout_a02b
+  INFO 2124WW
+  Thoendel
+  02-02-2020
+*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "wine", "vodka", "gin");        
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    if (text == 'Q' || text == 'q') {
+        clearScreen();
+        console.log("Bye");
+        process.exit();
+    } else if (text == 'V' || text == 'v'){
+        clearScreen();
+        viewList();
+        getPrompt();
+    }
+    else {
+        clearScreen();
+        checkitem(text);
+        if (Boolean === false) {

```

Comment:
not sure where Boolean is coming from. I would expect:
if (checkitem(text) == false){...do stuff